content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst

```diff
@@ -4,128 +4,141 @@ Process receipts and deliveries in one step
+
+.. note::
+   Since one-step receipt and delivery is the default for incoming and outgoing shipments in Odoo,
+   the *Multi-Step Routes* feature is *not* required.
+
+   However, for the above settings to appear on a warehouse form, the feature **must** be enabled.
+
+   To enable *Multi-Step Routes*, navigate to :menuselection:`Inventory app --> Configuration -->
+   Settings`. Under the :guilabel:`Warehouse` section, tick the checkbox next to
+   :guilabel:`Multi-Step Routes`, and click :guilabel:`Save`. Doing so also activates the
+   :guilabel:`Storage Locations` feature.
 
 Receive goods directly (1 step)
 ===============================
 
-Create a purchase order
------------------------
+When products are received in one step, they will move from the vendor location to warehouse stock
+in the database immediately upon validation of a purchase order (PO).
 
-On the main :menuselection:`Purchase` application dashboard, start by making a new quote by clicking
-:guilabel:`New`. Then, select (or create) a :guilabel:`Vendor` from the drop-down field, add a
-storable :guilabel:`Product` to the order lines, and click :guilabel:`Confirm Order` to finalize the
-quote as a new purchase order.
+Create purchase order
+---------------------
 
-A :guilabel:`Receipt` smart button will appear in the top-right corner of the :abbr:`PO (purchase
-order)` form — click it to reveal the associated receipt for the purchase order.
+To create a |PO|, navigate to the :menuselection:`Purchase app`, and click :guilabel:`New`. This
+opens a blank :guilabel:`Request for Quotation` (RfQ) form.
 
-.. image:: receipts_delivery_one_step/one-step-po-receipt.png
-   :align: center
-   :alt: Receipt smart button appears on the confirmed purchase order.
+Add a vendor in the :guilabel:`Supplier` field. Then, fill out the various fields on the |RfQ|, as
+necessary.
 
-.. tip::
-   Purchase order receipts can also be found in the :menuselection:`Inventory` application. In
-   the :guilabel:`Overview` dashboard, click the :guilabel:`# to Process` smart button in the
-   :guilabel:`Receipts` kanban card.
+.. image:: receipts_delivery_one_step/receipts-delivery-one-step-new-rfq.png
+   :align: center
+   :alt: Filled out new request for quotation form.
 
-   .. image:: receipts_delivery_one_step/one-step-to-process-btn.png
-      :align: center
-      :alt: Receipt kanban card's 1 to Process smart button.
+Under the :guilabel:`Products` tab, click :guilabel:`Add a product`, and select a product to add to
+the |RfQ|.
 
-Process the receipt
--------------------
+Once ready, click :guilabel:`Confirm Order`. This moves the |RfQ| to the :guilabel:`Purchase Order`
+stage.
 
-When viewing the receipt (associated with the purchase order above), click :guilabel:`Validate` to
-then complete the receipt.
+Once the |PO| is confirmed, a :guilabel:`Receipt` smart button appears at the top of the form.
+Clicking the smart button opens the warehouse receipt (WH/IN) form.
```

Comment:
I haven't been assigning it a guilabel since it's not actually written on the screen. But if I get this feedback again, I will implement it accordingly!